packages/toolpad-app/scripts/typings.mts

```diff
@@ -38,7 +38,7 @@ async function main() {
       LIBS.map(async ({ name }) => {
         const files: { filename: string; moduleId: string }[] = [];
         const resolvedPkg = require.resolve(`${name}/package.json`);
-        const pkgJson = await import(resolvedPkg, { assert: { type: 'json' } });
+        const pkgJson = await import(url.pathToFileURL(resolvedPkg).toString(), { assert: { type: 'json' } });
```

Comment:
Not sure what benefit the `import()` brings us. I'm fine with using `readJsonFile ` as well

Reply:
Okay - I found similar issues and fixes to this one in other libraries, I'll probably post them here too once the PR is ready, this seems like a weird issue.

Reply:
This was the issue here I think: https://github.com/nodejs/node/issues/31710